### DISCUSSION TITLE
Problem with create playlist via API

### DISCUSSION THREAD

**Discussion by honoluluman2** [AzuraCast/AzuraCast#5990] [Q&A]
On my WordPress installation I am using the following body values to send via API along with wp_remote_post in order to create an empty playlist.

$body = [
        'id'  => $playlist_ID,
	'name' => $playlist_title,
	'type'  => 'default',
	'source' => 'songs',
	'order'  => 'random',
	'remote_url' => null,
	'remote_type'  => 'stream',
	'remote_buffer'  => 0,
	'is_enabled' => false,
	'is_jingle'  => false,
	'play_per_songs' => 0,
	'play_per_minutes' => 0,
	'play_per_hour_minute'  => 0,
	'weight' => 5,
	'is_jingle'  => false,
	'include_in_requests' => false,
	'include_in_on_demand' => false,
	'backend_options' => [ '0' => ''],
	'avoid_duplicates' => true,
	'schedule_items' => [],
	'num_songs' => 0,
	'total_length' => 0,
];

the new playlist is being created with success but for some reason the 'id' value is not the one from $playlist_ID. Does anybody know why this is happening? It keeps creating its own value and i would need to pass my value.

**Comment by BusterNeece** [comment]
@honoluluman2 This is not possible via our API because our playlist IDs are auto-incrementing fields in the database. You will get a response back indicating the ID that was assigned in the database and should use that to track the newly created playlist in your own application.
